.zenodo.json

```diff
@@ -757,7 +820,12 @@
         {
             "name": "Ratul Das",
             "orcid": "0000-0002-5845-9979",
-            "affiliation": "National Institute of Science Education and Research, Bhubaneswar, Odisha, India"
+            "affiliation": "National Institute of Science Education and Research, Bhubaneswar"
+        },
+        {
+            "name": "Rehan Chalana",
+            "orcid": "0009-0005-1579-5787",
```

Comment:
Hi @RehanChalana, this is the same ORCID as Piyush Sharma, do you happen to have a your ORCID?